tools/cppcheck/suppressions-list

```diff
@@ -2,4 +2,4 @@ wrongmathcall:tests/unit-libm/test-libm.inc.h
 variableScope:jerry-libm/*.c
 invalidPointerCast:jerry-libm/*.c
 arithOperationsOnVoidPointer:jerry-libc/*.c
-commaSeparatedReturn:jerry-ext/include/jerryscript-ext/arg.impl.h
+commaSeparatedReturn:*
```

Comment:
Cppcheck warns on every struct literal in returns, because it misinterprets them as comma-separated expressions. This is quite bad IMO and obviously a shortcoming of cppcheck, because struct literals are completely valid C constructs.